gravitee-apim-console-webui/src/management/api/api-navigation/api-v4-menu.service.ts

```diff
@@ -372,16 +373,24 @@ export class ApiV4MenuService implements ApiMenuService {
     };
   }
 
-  private addApiTrafficMenuEntry(hasTcpListeners: boolean): MenuItem {
+  private addApiTrafficMenuEntry(hasTcpListeners: boolean, apiType: ApiType): MenuItem {
     if (this.permissionService.hasAnyMatching(['api-analytics-r'])) {
-      return {
-        displayName: 'API Traffic',
-        icon: 'bar-chart-2',
-        routerLink: hasTcpListeners ? 'DISABLED' : 'v4/analytics',
-        header: {
-          title: 'API Traffic',
-        },
-      };
+      if (apiType === 'PROXY') {
+        return {
+          displayName: 'API Traffic',
+          icon: 'bar-chart-2',
+          routerLink: hasTcpListeners ? 'DISABLED' : 'v4/analytics',
+        };
+      } else {
+        return {
+          displayName: 'API Traffic',
+          icon: 'bar-chart-2',
+          routerLink: hasTcpListeners ? 'DISABLED' : 'v4/analytics',
+          header: {
+            title: 'API Traffic',
+          },
+        };
+      }
```

Comment:
```suggestion
      const baseMenuItem = {
          displayName: 'API Traffic',
          icon: 'bar-chart-2',
          routerLink: hasTcpListeners ? 'DISABLED' : 'v4/analytics',
        };
      if (apiType === 'PROXY') {
        return baseMenuItem;
      } else {
        return {
          ...baseMenuItem,
          header: {
            title: 'API Traffic',
          },
        };
      }
```
WDYT to do this in order to help with future iterations and consistency between API Traffic pages?